src/ResourceManager/Compute/Commands.Compute/Extension/AzureDiskEncryption/Scripts/AzureDiskEncryptionPreRequisiteSetup.ps1

```diff
@@ -72,7 +72,7 @@ $ErrorActionPreference = "Stop"
         if(-not $SvcPrincipals)
```

Comment:
Each change to a module need to be accompanied by a change in the corresponding local changelog, to trigger our update and release of the module.

Reply:
done